pkg/proxy/service/cluster_service.go

```diff
@@ -56,7 +56,7 @@ func (s *ServiceImpl) GetNamespace(ctx *gin.Context, userID string) (*namespace.
 		return nil, errs.New("no member clusters found")
 	}
 	for _, member := range members {
-		if member.APIEndpoint == signup.APIEndpoint {
+		if member.APIEndpoint == signup.APIEndpoint && member.Name == signup.ClusterName {
```

Comment:
I guess you need it because of the e2e test env when we actually have two members in the same physical cluster?
Let's add a comment here. Because it might be unclear why just checking the API Endpoint is not enough.

Reply:
Done: c3fa10f2df514a1f696c726ec0eaf35086046b8a